R/aus_seasons.R

```diff
@@ -0,0 +1,47 @@
+library(lubridate)
```

Comment:
need to remove `library(lubridate)` from the file. Instead add `#' imports lubridate` to the roxygen header.

Reply:
Thanks for noticing the "leftover" `library(lubridate)`.  I've corrected it now.